src/transformers/generation_utils.py

```diff
@@ -958,13 +972,10 @@ def generate(
             remove_invalid_values=remove_invalid_values,
         )
 
-        stopping_criteria = self._get_stopping_criteria(max_length=max_length, max_time=max_time)
-        if max_length is not None:
```

Comment:
This shouldn't have been there before neither no?

Reply:
No it was fixed in another PR, it should be removed after rebase.